### PR TITLE
LPD-21422 [Marketplace] Remove Manage button and Add kebab

### PR DESCRIPTION
**Pull request by joanagrego**
Related to --> [LPD-19904](https://liferay.atlassian.net/browse/LPD-19904)

### CODE REVIEW

**Comment by joanagrego** [comment]
ci:test:sf

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/Table/TableButtons/TableKebabButton.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	>
+		<ClayDropDown.ItemList>
+			<ClayDropDown.Item onClick={onViewDetails}>
+				View Details
+			</ClayDropDown.Item>
+
+			<ClayDropDown.Item disabled={disabled} onClick={onEdit}>
+				Edit
+			</ClayDropDown.Item>
+		</ClayDropDown.ItemList>
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/Table/TableButtons/TableKebabButton.tsx:38]: Make this generic

**Review by kevenleone — CHANGES_REQUESTED** [review]

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/Table/TableButtons/TableKebabButton.tsx (hunk 1 above)
Comment (joanagrego) [reply workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/Table/TableButtons/TableKebabButton.tsx:38]: done

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Comment by joanagrego** [comment]
Resent --> [PR](https://github.com/brianchandotcom/liferay-portal/pull/148508)

**Pull request closed without merging**